CMakeLists.txt

```diff
@@ -3,76 +3,43 @@
 
 # Maintainer: mphoward
 
-# check for internal hoomd project
-set(HOOMD_INTERNAL OFF)
-if (PROJECT_NAME STREQUAL "HOOMD")
-    set(HOOMD_INTERNAL ON)
-else()
-    project(neighbor)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.9 FATAL_ERROR)
 
-    # set cmake policies consistent with HOOMD
-    CMAKE_MINIMUM_REQUIRED(VERSION 2.8.0 FATAL_ERROR)
-    if(CMAKE_MAJOR_VERSION VERSION_GREATER 2)
-        if(COMMAND cmake_policy)
-        cmake_policy(SET CMP0003 NEW)
-        cmake_policy(SET CMP0042 NEW)
-        endif(COMMAND cmake_policy)
-    endif(CMAKE_MAJOR_VERSION VERSION_GREATER 2)
+IF(NOT NEIGHBOR_HOOMD_INTERNAL)
+    # check for internal hoomd project
+    set(HOOMD_INTERNAL OFF)
 
-    # find HOOMD
-    include(FindHOOMD.cmake)
-    # try to deduce the hoomd version, since FindHOOMD does not support versioning now.
-    set(find_hoomd_version "
-from __future__ import print_function
-import sys,os
-sys.stdout = open(os.devnull, 'w')
-import hoomd
-print(hoomd.__version__, file=sys.stderr, end='')")
-    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "${find_hoomd_version}" ERROR_VARIABLE hoomd_version)
-    if(hoomd_version MATCHES "^Traceback*")
-        message(STATUS "HOOMD version: not deduced, ensure that it is >= 2.4.1.")
-    else()
-        message(STATUS "HOOMD version: " ${hoomd_version})
-        if(hoomd_version VERSION_LESS "2.4.1")
-        message(FATAL_ERROR "HOOMD version must be >= 2.4.1")
-        endif(hoomd_version VERSION_LESS "2.4.1")
-    endif(hoomd_version MATCHES "^Traceback*")
+    # Name the plugin project
+    project(neighbor LANGUAGES C CXX CUDA)
 
-    # plugins must be built as shared libraries
-    if(ENABLE_STATIC)
-        message(SEND_ERROR "Plugins cannot be built against a statically compiled hoomd")
-    endif(ENABLE_STATIC)
-    set(BUILD_SHARED_LIBS on)
+    # Find the installed HOOMD
+    find_package(HOOMD 2.6.0 REQUIRED)
```

Comment:
I'm assuming the version is 2.6 here because that's what `next` branched from? I'll remember to update this to 3.0 later.

Reply:
yes, I don't know when @joaander updates the `HOOMD_VERSION_RAW` in HOOMD's CMakeLists.txt so I am leaving it untouched for now.